pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -331,7 +319,7 @@ interface StatusCodeProps {
 
 const StatusCode: React.FC<StatusCodeProps> = ({ code, message = null }) => {
   return (
-    <span>{message} {code} <i className={"fas fa-circle logTable-statusCode " + (code > 199 && code < 300 ? 'logTable-statusCode--green' : 'logTable-statusCode--red')} ></i></span>
+    <span><i className={"fas fa-circle logTable-statusCode " + (code > 199 && code < 300 ? 'logTable-statusCode--green' : 'logTable-statusCode--red')} ></i> {code} {'(' + message + ')'}  </span>
```

Comment:
Couple of tips here:

1. The with the `` syntax you can write strings and variables without having to `+` everything, like so (this renders the exact same thing, improves the readability a lot): 

```
{`${code} (${message})`}
```

2. We have now implemented the package `@fortawesome/fontawesome-free`, meaning we can implement the fontawesome icons the _React_ way, instead of working with an `<i>` tag and adding the classes manually, like so:

```
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faCircle } from "@fortawesome/free-solid-svg-icons";
...
<FontAwesomeIcon className={`logTable-statusCode--${statusClassSuffix}`} icon={faCircle} />
```

3. This one line of code is getting really long and complex, partially due to the status code check within the class definition, a way to solve this would be (e.g.) to add a variable before the return statement:

```
const statusClassSuffix = (code > 199 && code < 300) ? "green" : "red"
... // see how we can implement this class in (2.)
```

Reply:
Code has been updated